src/css/job-posts.css

```diff
@@ -0,0 +1,299 @@
+.job-posts {
+  background:var(--COLOR_DARK_BLUE) center;
+}
+
+.job-posts__container {
+  padding-top: 95px;
+  padding-bottom: 70px;
+}
+
+.job-posts__title {
+  margin-bottom: 50px;
+  font-weight: var(--FONT_WEIGHT_700);
+  color: var(--COLOR_WHITE);
+  font-size: var(--FONT_SIZE_48);
+  letter-spacing: 0.05em;
+  line-height: 75px;
+}
+
+.job-posts__card-wrapper {
+  display: flex;
+  margin-bottom: 60px;
+  gap: 30px;
+  justify-content: center;
+}
+
+.card {
+  width: 370px;
+  height: 250px;
+  border-radius: 20px;
+  padding: 15px 30px 20px 20px;
+  background: var(--COLOR_WHITE);
+}
+
+.card:first-child {
+  background: rgba(238, 79, 60, 0.5);
+}
+
+.card-content__top {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.card-content__left-side {
+  display: flex;
+  align-items: center;
+}
+
+.card-content__card-circle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 55px;
+  width: 55px;
+  border-radius: 50px;
+  background-color: var(--COLOR_WHITE);
+  margin-right: 20px;
+  box-shadow: 0px 4px 50px rgba(153, 153, 153, 0.25);
```

Comment:
I think it is reasonable to move the color to CSS variables and use the variable here